android: replay faq (#11247)
getsentry/sentry-docs: docs/platforms/android/session-replay/index.mdx

- The replay was rate-limited and couldn't be accepted.
- The replay was deleted by a member of your org.
- There were network errors and the replay wasn't saved.

## FAQ

Q: Doesn't it work with Jetpack Compose?
A: Yes. But please be aware we're still adding support for redaction for Jetpack Compose views. This will be added during the Beta program so after GA these views will be masked by default.

Q: What's the lowest version of Android supported?

> A: The same version supported by the SDK itself. There's no additional requirement specific for Replay

Author: ```suggestion
A: Recording only happens on Android 8 (API level 26) or newer. On devices running an older version, other SDK features work normally, but no recording will be performed.
```

> 
> Q: Doesn't it work with Jetpack Compose?
> A: Yes. But please be aware we're still adding support for redaction for Jetpack Compose views. This will be added during the Beta program so after GA these views will be masked by default.
> 
> Q: What's the lowest version of Android supported?
> A: Recording only happens on Android 8 (API level 26) or newer. On devices running an older version, other SDK features work normally, but no recording will be performed.

Reviewer: ```suggestion

Q: Does Session Replay work with Jetpack Compose?
A: Yes, but we're still adding support for redaction for Jetpack Compose views. Redaction will be added during the Beta program and after GA these views will be masked by default.

Q: What's the lowest version of Android supported?
A: Recording only happens on Android 8 (API level 26) or newer. For devices running an older version, SDK features other than recording work normally.
```
There are no line breaks between Q/A in the preview unfortunately.
